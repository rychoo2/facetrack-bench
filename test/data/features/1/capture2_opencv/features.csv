filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.38125,0.029166666666666667,0.209375,0.37222222222222223,0.5,0.6828358208955224,0.4962686567164179,0.1791044776119403,0.04477611940298507,0.2126865671641791,0.7350746268656716,0.30970149253731344,0.585820895522388,-0.029850746268656716,0.2462686567164179,0.34701492537313433,0.45714285714285713,0.5303030303030303,0.4153846153846154,0.5454545454545454
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,0.425,0.05694444444444444,0.2515625,0.44722222222222224,0.562111801242236,0.6770186335403726,0.5,0.2826086956521739,0.08695652173913043,0.3385093167701863,0.6987577639751553,0.32608695652173914,0.531055900621118,-0.06832298136645963,0.2919254658385093,0.38819875776397517,0.5423728813559322,0.3888888888888889,0.4714285714285714,0.5333333333333333
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.41171875,0.041666666666666664,0.2609375,0.4638888888888889,0.5718562874251497,0.655688622754491,0.5,0.24550898203592814,0.10179640718562874,0.2934131736526946,0.718562874251497,0.3143712574850299,0.5389221556886228,-0.05089820359281437,0.3203592814371258,0.36826347305389223,0.5748031496062992,0.4339622641509434,0.5069444444444444,0.5102040816326531
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,0.428125,0.03888888888888889,0.25703125,0.45694444444444443,0.5349544072948328,0.6413373860182371,0.49544072948328266,0.2735562310030395,0.11854103343465046,0.3252279635258359,0.6899696048632219,0.3191489361702128,0.5045592705167173,-0.06382978723404255,0.2948328267477204,0.3738601823708207,0.5423728813559322,0.375,0.4566929133858268,0.48484848484848486
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,0.41875,0.03194444444444444,0.27421875,0.4875,0.5584045584045584,0.6410256410256411,0.50997150997151,0.28205128205128205,0.150997150997151,0.3418803418803419,0.6866096866096866,0.32193732193732194,0.4757834757834758,-0.06267806267806268,0.32763532763532766,0.37606837606837606,0.5299145299145299,0.3888888888888889,0.4881889763779528,0.46153846153846156
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,,,,,,,,,,,,,,,,,,,,
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,,,,,,,,,,,,,,,,,0.6373626373626373,0.1794871794871795,0.4881889763779528,0.46153846153846156
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,,,,,,,,,,,,,,,,,0.6304347826086957,0.30434782608695654,0.5036496350364964,0.4782608695652174
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.36875,0.043055555555555555,0.26171875,0.4652777777777778,0.5014925373134328,0.7044776119402985,0.47761194029850745,0.29850746268656714,0.10746268656716418,0.33134328358208953,0.6746268656716418,0.3492537313432836,0.5343283582089552,-0.050746268656716415,0.2746268656716418,0.37910447761194027,0.46153846153846156,0.4722222222222222,0.42748091603053434,0.4444444444444444
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.38828125,0.05416666666666667,0.25078125,0.44583333333333336,0.4766355140186916,0.7009345794392523,0.45482866043613707,0.2803738317757009,0.04361370716510903,0.3115264797507788,0.6728971962616822,0.3395638629283489,0.5264797507788161,-0.04672897196261682,0.24610591900311526,0.37694704049844235,0.48951048951048953,0.475,0.45454545454545453,0.525
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.4140625,0.06944444444444445,0.24375,0.43333333333333335,0.5128205128205128,0.7051282051282052,0.46153846153846156,0.2692307692307692,0.04487179487179487,0.33653846153846156,0.6891025641025641,0.3141025641025641,0.5288461538461539,-0.07692307692307693,0.25,0.3685897435897436,0.5298507462686567,0.3888888888888889,0.45714285714285713,0.37037037037037035
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.39609375,0.07083333333333333,0.23203125,0.4125,0.5016835016835017,0.6902356902356902,0.468013468013468,0.2828282828282828,0.037037037037037035,0.32323232323232326,0.7205387205387206,0.32996632996632996,0.5757575757575758,-0.050505050505050504,0.2558922558922559,0.37037037037037035,0.5244755244755245,0.3888888888888889,0.4642857142857143,0.42424242424242425
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,,,,,,,,,,,,,,,,,,,,
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.3921875,0.022222222222222223,0.2109375,0.375,0.5037037037037037,0.6518518518518519,0.4777777777777778,0.1814814814814815,0.011111111111111112,0.21851851851851853,0.7296296296296296,0.29259259259259257,0.5962962962962963,-0.037037037037037035,0.23703703703703705,0.32592592592592595,0.46258503401360546,0.5084745762711864,0.42657342657342656,0.4915254237288136
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,,,,,,,,,,,,,,,,,0.45,0.5362318840579711,0.3884297520661157,0.5072463768115942
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.37890625,0.059722222222222225,0.2296875,0.4083333333333333,0.46938775510204084,0.6768707482993197,0.46258503401360546,0.30272108843537415,0.11224489795918367,0.3401360544217687,0.6904761904761905,0.3401360544217687,0.54421768707483,-0.034013605442176874,0.3435374149659864,0.3843537414965986,0.48905109489051096,0.3333333333333333,0.576271186440678,0.43333333333333335
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.4,0.025,0.21640625,0.38472222222222224,0.5018050541516246,0.6570397111913358,0.48736462093862815,0.1407942238267148,0.01444043321299639,0.18050541516245489,0.7364620938628159,0.26353790613718414,0.6534296028880866,-0.039711191335740074,0.24548736462093862,0.30685920577617326,0.4928571428571429,0.5151515151515151,0.4444444444444444,0.5303030303030303
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,0.3875,0.04861111111111111,0.2046875,0.3638888888888889,0.5152671755725191,0.683206106870229,0.4847328244274809,0.29389312977099236,0.08778625954198473,0.3244274809160305,0.7290076335877863,0.33587786259541985,0.5648854961832062,-0.030534351145038167,0.22900763358778625,0.36259541984732824,0.5470085470085471,0.3055555555555556,0.3162393162393162,0.30303030303030304
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.38359375,0.03611111111111111,0.21328125,0.37916666666666665,0.4835164835164835,0.6483516483516484,0.48717948717948717,0.2087912087912088,0.054945054945054944,0.25274725274725274,0.7142857142857143,0.315018315018315,0.5677655677655677,-0.02564102564102564,0.23809523809523808,0.34798534798534797,0.44285714285714284,0.5178571428571429,0.3937007874015748,0.49056603773584906
