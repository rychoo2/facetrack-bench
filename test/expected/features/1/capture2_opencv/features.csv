filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_eye_distance_x,rel_eye_distance_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.38125,0.029166666666666667,0.209375,0.37222222222222223,0.5,0.6828358208955224,0.585820895522388,-0.029850746268656716,0.45714285714285713,0.5303030303030303,,
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,0.425,0.05694444444444444,0.2515625,0.44722222222222224,0.562111801242236,0.6770186335403726,0.531055900621118,-0.06832298136645963,0.5423728813559322,0.3888888888888889,0.4714285714285714,0.5333333333333333
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.41171875,0.041666666666666664,0.2609375,0.4638888888888889,0.5718562874251497,0.655688622754491,0.5389221556886228,-0.05089820359281437,0.5748031496062992,0.4339622641509434,0.5138888888888888,0.5306122448979592
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,0.428125,0.03888888888888889,0.25703125,0.45694444444444443,0.5349544072948328,0.6413373860182371,0.5045592705167173,-0.06382978723404255,,,0.4566929133858268,0.48484848484848486
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,0.41875,0.03194444444444444,0.27421875,0.4875,0.5584045584045584,0.6410256410256411,0.4757834757834758,-0.06267806267806268,0.5299145299145299,0.3888888888888889,,
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,,,,,,,,,,,,
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,,,,,,,,,0.6373626373626373,0.1794871794871795,0.4881889763779528,0.4358974358974359
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,,,,,,,,,0.6304347826086957,0.30434782608695654,0.5109489051094891,0.4782608695652174
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.36875,0.043055555555555555,0.26171875,0.4652777777777778,0.5014925373134328,0.7044776119402985,0.5343283582089552,-0.050746268656716415,0.46153846153846156,0.4722222222222222,0.42748091603053434,0.4444444444444444
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.38828125,0.05416666666666667,0.25078125,0.44583333333333336,0.4766355140186916,0.7009345794392523,0.5264797507788161,-0.04672897196261682,0.4825174825174825,0.475,0.45454545454545453,0.525
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.4140625,0.06944444444444445,0.24375,0.43333333333333335,0.5128205128205128,0.7051282051282052,0.5288461538461539,-0.07692307692307693,0.5298507462686567,0.3888888888888889,0.45714285714285713,0.37037037037037035
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.39609375,0.07083333333333333,0.23203125,0.4125,0.5016835016835017,0.6902356902356902,0.5757575757575758,-0.050505050505050504,0.5244755244755245,0.3888888888888889,,
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,,,,,,,,,,,,
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.3921875,0.022222222222222223,0.2109375,0.375,0.5037037037037037,0.6518518518518519,0.5962962962962963,-0.037037037037037035,0.46258503401360546,0.5084745762711864,,
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,,,,,,,,,0.45,0.5362318840579711,0.3884297520661157,0.5072463768115942
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.37890625,0.059722222222222225,0.2296875,0.4083333333333333,0.46938775510204084,0.6768707482993197,0.54421768707483,-0.034013605442176874,0.48905109489051096,0.3333333333333333,,
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.4,0.025,0.21640625,0.38472222222222224,0.5018050541516246,0.6570397111913358,0.6534296028880866,-0.039711191335740074,0.4857142857142857,0.5454545454545454,0.4444444444444444,0.5303030303030303
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,0.3875,0.04861111111111111,0.2046875,0.3638888888888889,0.5152671755725191,0.683206106870229,0.5648854961832062,-0.030534351145038167,0.5470085470085471,0.3055555555555556,0.3162393162393162,0.30303030303030304
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.38359375,0.03611111111111111,0.21328125,0.37916666666666665,0.4835164835164835,0.6483516483516484,0.5677655677655677,-0.02564102564102564,0.44285714285714284,0.5178571428571429,0.3937007874015748,0.49056603773584906
